src/Sylius/Bundle/AddressingBundle/Form/Type/ProvinceCodeChoiceType.php

```diff
@@ -54,7 +54,7 @@ public function getName()
      *
      * @return array
      */
-    private function getProvinceCodes(array $provinces)
+    private function getProvinceCodes($provinces)
```

Comment:
In case of [this line](https://github.com/ravaj-group/Sylius/blob/fix/province-bugs/src/Sylius/Bundle/AddressingBundle/Form/Type/ProvinceCodeChoiceType.php#L34) `provinces` is a doctrine Collection not array.